docs/en/sql-reference/functions/other-functions.md

```diff
@@ -794,6 +794,36 @@ Result:
 └────────────────┴────────────┘
 ```
 
+## fromReadableSize
```

Comment:
Let's add docs for the other function as well.

Reply:
Added docs for all flavours of `fromReadable*Size*`